torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1324,18 +1329,31 @@ def wrapper(self, *args, **kwargs):
     wrapper.cache_info = fn_cache.cache_info  # type: ignore[attr-defined]
     return wrapper
 
+@dataclass(frozen=True)
+class ShapeGuardExprSources:
+    """
+    Represents a shape guard expression, with the related sources.
+
+    Ex:
+        expr: s0 < s1 * 2
```

Comment:
Is the expr really s0/s1? It seems to me like you ought to have gotten x < y * 2 here?